src/sentry/api/serializers/models/group.py

```diff
@@ -1040,6 +1053,16 @@ def _seen_stats_generic(
             bool(self.start or self.end or self.conditions),
             self.environment_ids,
         )
+        lifetime_result = self._parse_seen_stats_results(
+            self._execute_generic_seen_stats_query(
+                item_list=generic_issue_list,
+            ),
+            generic_issue_list,
+            False,
+        )
+        for item in generic_issue_list:
+            result[item].update({"lifetime": lifetime_result.get(item, {})})
```

Comment:
Why do we still make a query for lifetime stats here? You should be able to use the same global function